packages/core/src/external/opensearch/search-documents.ts

```diff
@@ -54,7 +57,8 @@ function checkExtensions(doc: DocumentReference) {
   if (!extensions) return false;
   const metriport = extensions.find(isMetriportExtension);
   const cw = extensions.find(isCommonwellExtension);
-  if (!metriport && !cw) return false;
+  const cq = extensions.find(isCarequalityExtension);
+  if (!metriport && !cw && !cq) return false;
```

Comment:
BUG! search-dcouments.ts code exists in core and in api, and it was only updated in api causing all CQ documents to not come up in search